assets/js/rolltopsack.js

```diff
@@ -38,8 +38,8 @@ calculatorSetup(function () {
     bottomLength,
     bottomWidth,
     sA,
```

Comment:
seam allowance is returned in decimal form "0.625" which is correct. However previous implementation returned 5/8 which is easier to read within the instructions. Possible to return as fractional rather than decimal?

Reply:
Do you want all imperial measurements to return as a fraction or only that specific one? I'll think on it a bit. Most people know fractions for 1/2 and 1/4s but 8ths are harder

On the current site the Cinch Sack calculator currently returns with a decimal but the Roll-Top Stuff Sack returns as a fraction.

Reply:
Reviewing makes me question a few of my choices haha. As you said the 1/8" ones are tough to read in decimal form. Seam allowance specifically is likely the one measurement that makes most sense to have fractional since the 3/8" and  5/8" are very commonly used. 3/8" or 1cm is the most common but different seam types call for different allowances. The roll top sack uses a french seam so it has the 5/8" seam allowance. I noticed I used 1/2" seam allowances on some generators which I'm guessing I did for readability in the tutorial write up. 

The roll top was the latest generator and at the time I was considering making a selectable seam allowance. But I figured beginners would be confused so just left it as a fixed seam type. In this case, I could just as easily go back to simply stating the seam allowance rather than making dynamic.

Reply:
See what you think of it with fractions. There might need to be some updates to the static part of the site to make everything match if you like the way fractions looks.

Reply:
This is great! I think the readability of all dimension in fractional form is much easier and more straightforward.